### DISCUSSION TITLE
[Panel] Creating panel of presumable category ticket agents

### DISCUSSION THREAD

**Discussion by mwpowellhtx** [discord-tickets/bot#487] [Support (Q&A)]
Prompted by:

```
Error
Missing Access
```

Followed by:

```
 Make sure members can read but not send messages in #server-help.
```

As far as I know the bot has access to the desired help channel. Also as far as I am aware, **@everyone** access to the channel has been stripped away in terms of the indicated messaging access. But still I am getting this.

**Comment by eartharoid** [comment]
@mwpowellhtx If you're self-hosting there should be an error in the bot's console with more information.
When creating a panel, Missing Access could mean the bot
- doesn't have permission to create the channel
- can't view the existing channel
- can't send messages in the channel
- can't alter the channel permissions.

To fix permissions problems, make sure you add the bot using the generated URL (by clicking the `Add` button on the server selector page of the settings panel) and move the bot's role to the top of the role list.

**Reply by mwpowellhtx** [reply]
I followed the URL, AFAIK, that's how it landed in my discord. I'll check the bot has some perms, though. To clarify, that's in the bot `ROLES & MEMBERS`, correct? was in **@everyone**, added my admin role for that.

**Reply by eartharoid** [reply]
Yes, in `Discord > Server Settings > Roles` there should be a role that has the same name as your bot.  Make sure it has these permissions and drag it to the top.

![image](https://github.com/discord-tickets/bot/assets/20905071/3decb6b5-a1ed-4b7e-8b3e-aa8ba1045cde)

**Reply by mwpowellhtx** [reply]
A role same name as my bot? Nope. Not even close. But I'll look into this part, thank you.

**Reply by mwpowellhtx** [reply]
Now getting this.

```
Error
Missing Permissions
```

```
 Make sure members can read but not send messages in #server-help.
```

I mean, I'm stripping members of everything, it seems like, but for _Read Message History_.

**Reply by mwpowellhtx** [reply]
Getting back an error in the browser:
```
403 (Forbidden)
```

**Reply by mwpowellhtx** [reply]
Tickets added to both the _help_ and corresponding _log_ channels. Does the _message_ thing also apply to the _log_ channel?

**Reply by eartharoid** [reply]
```
 Make sure members can read but not send messages in #server-help.
```
This is just a hint and incorrect permissions for your member won't affect the bot's ability to create the panel. The problem is that the bot is missing permission or isn't high enough in the role hierarchy.

Join https://lnk.earth/discord and DM me.

**Comment by mwpowellhtx** [comment]
I must be a complete dunce at discord, or I do not know what the message means send messages, or which channel(s) it is talking about. Could I trouble you to visit my discord for a brief moment @eartharoid ?

**Reply by mwpowellhtx** [reply]
Thanks for the looksee and adjustments. 👍

**Comment by mwpowellhtx** [comment]
Not sure it is a gap in the web UI or what, but the _panels_ is failing to enumerate existing panels. Again, perms are all there, does have _read message history_, for instance, educated guess.

Also, getting familiar with what to expect, folks submit a ticket, one opens in _logs_, I suspect (?), and someone on my team can _claim_ that to follow up? When satisfied, resolved, _close_ it out, along these lines?

Thanks! 🎆
